ui/src/app/shared/service/utils.ts

```diff
@@ -110,14 +110,18 @@ export class Utils {
    * @param v1 
    * @param v2 
    */
-  public static subtractSafely(v1: number, v2: number): number {
-    if (v1 == null) {
-      return v2;
-    } else if (v2 == null) {
-      return v1;
-    } else {
-      return v1 - v2;
-    }
+  public static subtractSafely(...values: number[]): number {
+    var result = null;
+    values.forEach(v => {

```

Comment:
i think you could also filter out all null values before, would be better readable